inst/examples/joint_rpca_example.qmd

```diff
@@ -0,0 +1,122 @@
+---
+title: "Joint RPCA Reproducible Example"
+format: html
+editor: visual
+---
+
+```{r}
+options(warn = -1)
+# Load user-defined functions
+source("../R/joint_rpca.R")
+source("../R/joint_optspace_helper.R")
+source("../R/joint_optspace_solve.R")
+source("../R/optspace_helper.R")
+source("../R/transform_helper.R")
+source("../R/transform.R")
+source("../R/rclr_transform.R")
+source("../R/mask_value_only.R")
+source("../R/rpca_table_processing.R")
+source("../R/joint_rpca_utils.R")
```

Comment:
As soon as the package passes the tests this should assume that the mia version including these is installed (instead of separately reading these from files).